go.mod

```diff
@@ -1,40 +1,36 @@
 module github.com/hashicorp/packer-plugin-azure
 
-go 1.19
+go 1.21
```

Comment:
https://github.com/hashicorp/packer-plugin-azure/blob/36cc195a84e36f246666d375c1e334fd8a5ef5ed/.go-version#L1

Reply:
Is 1.21.8 a requirement for these changes or should we just always set to 1.21.N since this is technically the latest version?

Reply:
Not at all, I just bumped us to 1.21 based on what was already set in the .go-version, these changes should work independent of go minor version.